x-pack/plugins/ml/common/util/chart_utils.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CHART_TYPE, ChartType } from '../constants/charts';
+import type { SeriesConfigWithMetadata } from '../types/results';
+
+/**
+ * TODO check if these flags are still needed
```

Comment:
Did you add this `TODO`? Are these left over from the original work @walterra did to add the distribution charts? If so, I can't see why they are needed any more.

Reply:
Removed in 29f948f9ab1cff24844b62fc6d0b6225776bdef9